static/js/theme-map/ThemeMapConfig.js

```diff
@@ -24,6 +24,22 @@ export default class ThemeMapConfig {
      */
     this.apiKey = jsonConfig.apiKey;
 
+    /**
+     * Controls the visual offset of each pin.
```

Comment:
nit: do we want the @type here too?

Reply:
I feel like you can tell it's an object by looking at it but I'll add for consistency